README.md

```diff
@@ -22,35 +22,49 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
 # Contents
 
 - [What is Docker?](#what-is-docker)
-- [Where to start?](#where-to-start)
-- [Where to start? (Windows)](#where-to-start-windows)
-- [Tools](#tools)
-	- [Continuous Integration / Continuous Delivery](#continuous-integration--continuous-delivery)
-		- [CI Services](#ci-services)
-	- [Deployment and Infrastructure](#deployment-and-infrastructure)
-	- [Developer Tools](#developer-tools)
-	- [Development Environments](#development-environments)
-	- [Dockerfile](#dockerfile)
-	- [Garbage Collection](#garbage-collection)
-	- [Hosting Images (registries)](#hosting-images-registries)
-	- [Image Builder](#image-builder)
-	- [Linter / Validator](#linter--validator)
-	- [Local Container Manager](#local-container-manager)
-	- [Monitoring & Logging](#monitoring--logging)
-		- [Monitoring & Logging Services](#monitoring--logging-services)
-	- [Networking](#networking)
-	- [PaaS](#paas)
-	- [Remote Container Manager / Orchestration](#remote-container-manager--orchestration)
-	- [Reverse Proxy](#reverse-proxy)
-	- [Security](#security)
-	- [Serverless](#serverless)
-	- [Service Discovery](#service-discovery)
-	- [CaaS - Services for running containers](#services-for-running-containers)
-	- [Terminal User Interface](#terminal-user-interface)
-	- [Testing](#testing)
-	- [Utilities](#utilities)
-	- [Volume management and plugins](#volume-management-and-plugins)
-	- [Web Interface](#web-interface)
+	- [Where to start?](#where-to-start)
+	- [Where to start? (Windows)](#where-to-start-windows)
+- [Managing Containers](#)
```

Comment:
I think that "Container Operations" suggested by @vegasbrianc is a better description. What do you think?